src/services/users/user_functions.ts

```diff
@@ -0,0 +1,23 @@
+import {MSUser} from '../../model/MSUser';
+import {MSContact} from '../../model/MSContact';
+import {BookitUser} from '../../model/BookitUser';
+
```

Comment:
@mathomas I'm not sure what you mean. Are you saying we should test the models? What would that look like?